### DISCUSSION TITLE
Exporting high-quality images from Blender

### DISCUSSION THREAD

**Discussion by JAbrilGarrido** [BradyAJohnston/MolecularNodes#154] [Q&A]
Hi everyone!

I was wondering was the best way to export images/animations from Blender without losing any quality of the protein model. Is there any way to control the DPI? Essentially, after I render and save the image in any format (also .exr), the image is quite pixelated when zooming in.

Hope anyone can help!

Greetings,
Julio

**Comment by BradyAJohnston** [comment]
Hi mate! To save the image as a different size, you'll have to render the image at the different size first. Those setting are under scene properties, format.

![image](https://user-images.githubusercontent.com/36021261/218344827-17685f8b-568a-44b8-86ce-872d5c6e8378.png)
